### PR TITLE
Update ee-install.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Docs for: https://github.com/mattermost/mattermost-server/pull/14539.

### CODE REVIEW

File: source/install/ee-install.rst [hunk 1]
```diff
@@ -67,7 +67,108 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+
+Incompatible system architecture
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This tool is currently only compatible with Linux systems using x86-64 architecture. If you’re running Mattermost on a different architecture, please follow the manual upgrade process.
+
+Can’t get file
+^^^^^^^^^^^^^^^^
+
+If the upgrade fails due to file retrieval failure, unavailable binary, or connectivity error please check your proxy settings and try again. If the problem persists, follow the manual upgrade process instead.
```
Comment (amyblais) [inline source/install/ee-install.rst:171]:
```suggestion
If the upgrade fails due to file retrieval failure, unavailable binary, or connectivity error, please check your proxy settings and try again. If the problem persists, follow the manual upgrade process instead.
```

File: source/install/ee-install.rst [hunk 2]
```diff
@@ -67,7 +67,108 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+During the upgrade process, the Mattermost Enterprise Edition binary file that matches your version of Mattermost is downloaded, decompressed, extracted, and overwrites the Team Edition executable. Once this process is complete, you're prompted to restart your server. 
+
+The Mattermost version listed in **System Console > Edition and License** will change from **Team Edition** to **Enterprise Edition**, and you can activate an E20 trial.
+
+GitLab Omnibus
+~~~~~~~~~~~~~~
+
+If you’re using GitLab Omnibus to manage your Mattermost installation, you may encounter an error during the upgrade due to permission restrictions. This is resolved by changing the file permissions manually. Changing the permissions in this way doesn't affect your Mattermost deployment or impact any data. The permission change is done solely for the upgrade.
+
+To change the permissions using the command line on the Mattermost server you need access to the command line tool as *mattermost* user.
```
Comment (amyblais) [inline source/install/ee-install.rst:97]:
```suggestion
To change the permissions using the command line on the Mattermost server, you need access to the command line tool as *mattermost* user.
```

**Review by amyblais — APPROVED** [review]

File: source/install/ee-install.rst [hunk 3]
```diff
@@ -67,7 +67,108 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+Initiating the Conversion
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Navigate to **System Console > Edition and License** and select **Upgrade to Enterprise Edition**.
+
+During the upgrade process, the Mattermost Enterprise Edition binary file that matches your version of Mattermost is downloaded, decompressed, extracted, and overwrites the Team Edition executable. Once this process is complete, you're prompted to restart your server. 
+
+The Mattermost version listed in **System Console > Edition and License** will change from **Team Edition** to **Enterprise Edition**, and you can activate an E20 trial.
+
+GitLab Omnibus
```
Comment (justinegeffen) [inline source/install/ee-install.rst:92]:
```suggestion
Permissions and Limitations
```

File: source/install/ee-install.rst [hunk 4]
```diff
@@ -67,7 +67,108 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+~~~~~~~~~~~~~~
+
+If you’re using GitLab Omnibus to manage your Mattermost installation, you may encounter an error during the upgrade due to permission restrictions. This is resolved by changing the file permissions manually. Changing the permissions in this way doesn't affect your Mattermost deployment or impact any data. The permission change is done solely for the upgrade.
```
Comment (justinegeffen) [inline source/install/ee-install.rst:95]:
```suggestion
If you're using a package manager, such as GitLab Omnibus, to manage your Mattermost installation the Mattermost user won't have permissions to perform the upgrade. In this case you may encounter an error which is resolved by changing the file permissions manually. 

Changing the permissions in this way doesn't affect your Mattermost deployment or impact any data. The permission change is done solely for the upgrade.
```

File: source/install/ee-install.rst [hunk 5]
```diff
@@ -67,7 +67,91 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+~~~~~~~~~~~~~~~~
+
+Mattermost has reverted to Team Edition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you converted Team Edition to Enterprise Edition on a managed deployment and then upgraded, the upgrade will have overwritten Enterprise Edition with the latest version of Team Edition.
+
+You can convert to Enterprise Edition again by following the steps above. If you plan to use Mattermost Enterprise Edition permanently, we recommend migrating your server to a self-managed one.
+
+The manual process reset my file permissions. How do I get them back?
```
Comment (esadur) [inline source/install/ee-install.rst:120]:
```suggestion
The manual process reset my file permissions
```

File: source/install/ee-install.rst [hunk 6]
```diff
@@ -67,7 +67,91 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+
+If you manually changed your file permissions, you can change them back.
+
+1. Open the command line tool on the Mattermost server.
+2. `cd` to the Mattermost installation directory.
+3. Enter: 
+
+.. code-block:: none
+  
+  \n\n```\nchown {{.FileUsername}} \"{{.Path}}\"\nchmod -w \"{{.Path}}\"\n```"
```
Comment (esadur) [inline source/install/ee-install.rst:131]:
```suggestion
chown <OriginalFileOwner> <PathToBinaryFile>

chmod -w <PathToBinaryFile>
```

File: source/install/ee-install.rst [hunk 7]
```diff
@@ -67,7 +67,91 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+.. code-block:: none
+
+   \n\n```\nchown {{.MattermostUsername}} \"{{.Path}}\"\nchmod +w \"{{.Path}}\"\n```\n\
+
+Incompatible system architecture
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This tool is currently only compatible with Linux systems using x86-64 architecture. If you’re running Mattermost on a different architecture, please follow the manual upgrade process.
+
+Can’t get file
```
Comment (esadur) [inline source/install/ee-install.rst:151]:
```suggestion
Can’t retrieve Enterprise Edition binary file
```

**Review by esadur — DISMISSED** [review]

File: source/install/ee-install.rst [hunk 8]
```diff
@@ -67,7 +67,91 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+
+If you're using a package manager, such as GitLab Omnibus, to manage your Mattermost installation the Mattermost user won't have permissions to perform the upgrade. In this case you may encounter an error which is resolved by changing the file permissions manually. 
+
+Changing the permissions in this way doesn't affect your Mattermost deployment or impact any data. The permission change is done solely for the upgrade.
+
+To change the permissions using the command line on the Mattermost server, you need access to the command line tool as *mattermost* user. Open the command line tool on the Mattermost server, `cd` to the Mattermost installation directory, and run the following command.
+
+.. code-block:: none
+
+  \n\n```\nchown {{.MattermostUsername}} \"{{.Path}}\"\nchmod +w \"{{.Path}}\"\n```\n\
```
[the remark this answers is not in the document]
Comment (esadur) [reply source/install/ee-install.rst:100]: @jespino can you look at these commands and make sure they're correct? The commands from your PR had variable references to the system username and the path to the binary file. I believe that the system user needs to be `mattermost`, so I wrote that in.

File: source/install/ee-install.rst [hunk 9]
```diff
@@ -67,7 +67,91 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+The Mattermost version listed in **System Console > Edition and License** will change from **Team Edition** to **Enterprise Edition**, and you can activate an E20 trial.
+
+Permissions and Limitations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you're using a package manager, such as GitLab Omnibus, to manage your Mattermost installation the Mattermost user won't have permissions to perform the upgrade. In this case you may encounter an error which is resolved by changing the file permissions manually. 
+
+Changing the permissions in this way doesn't affect your Mattermost deployment or impact any data. The permission change is done solely for the upgrade.
+
+To change the permissions using the command line on the Mattermost server, you need access to the command line tool as *mattermost* user. Open the command line tool on the Mattermost server, `cd` to the Mattermost installation directory, and run the following command.
```
[the remark this answers is not in the document]
Comment (esadur) [reply source/install/ee-install.rst:96]: @jespino I wrote that the typical path to the binary file is `/opt/mattermost/bin/mattermost`. Is that correct?

File: source/install/ee-install.rst [hunk 10]
```diff
@@ -67,7 +67,95 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+
+.. code-block:: none
+
+  \n\n```\nchown {{.MattermostUsername}} \"{{.Path}}\"\nchmod +w \"{{.Path}}\"\n```\n\
+
+In the Mattermost System Console, retry the upgrade. When the upgrade is complete, return to the command prompt on the Mattermost server and run the following command to restore the file permissions, replacing <OriginalFileOwner> with the appropriate value:
```
Comment (justinegeffen) [inline source/install/ee-install.rst:102]:
```suggestion
In the Mattermost System Console, retry the upgrade. When the upgrade is complete, return to the command prompt on the Mattermost server and run the following command to restore the file permissions, replacing `<OriginalFileOwner>` with the appropriate value:
```

File: source/install/ee-install.rst [hunk 11]
```diff
@@ -52,11 +52,11 @@ GitLab Omnibus runs the open source Mattermost Team Edition. To upgrade to Matte
 
  - Go to ``/etc/gitlab/gitlab.rb`` and set the following line to false
 
-   .. code-block:: text
+.. code-block:: text
 
     mattermost['enable'] = false
 
- - Run `sudo gitlab-ctl reconfigure` to apply the updated setting
+- Run `sudo gitlab-ctl reconfigure` to apply the updated setting
```
Comment (amyblais) [inline source/install/ee-install.rst:59]: This bullet point is misaligned compared to the bullet point above ("Go to /etc/gitlab/gitlab.rb and set the following line to false")

File: source/install/ee-install.rst [hunk 12]
```diff
@@ -67,7 +67,100 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+Navigate to **System Console > Edition and License** and select **Upgrade to Enterprise Edition**.
+
+During the upgrade process, the Mattermost Enterprise Edition binary file that matches your current server version is downloaded, decompressed, and extracted. The Team Edition binary is then replaced by the Enterprise Edition version. Once this process is complete, you're prompted to restart your server.
+
+The Mattermost version listed in **System Console > Edition and License** will change from **Team Edition** to **Enterprise Edition**, and you can now activate an Enterprise Edition trial or upload a license.
+
+Permissions and Limitations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you're using a package manager, such as GitLab Omnibus, to manage your Mattermost installation, the Mattermost system user won't have sufficient permissions to perform the upgrade. If this is the case, you'll need to change the file permissions manually.
```
Comment (amyblais) [inline source/install/ee-install.rst:92]:
```suggestion
If you're using a package manager, such as GitLab Omnibus, to manage your Mattermost installation the Mattermost system user won't have sufficient permissions to perform the upgrade. If this is the case, you'll need to change the file permissions manually.
```

File: source/install/ee-install.rst [hunk 13]
```diff
@@ -67,7 +67,100 @@ If you need to migrate Team Edition prior to install, `please follow the migrati
...
+
+Changing the permissions in this way doesn't affect your Mattermost deployment or impact any data. The permission change is done solely for the upgrade.
+
+To change the permissions using the command line on the Mattermost server, you need access to the command line tool as *mattermost* user. 
+Open the command line tool on the Mattermost server and ``cd`` to the Mattermost installation directory. Run the following commands (replacing ``<PathToBinaryFile>`` with the appropriate path (typically ``/opt/mattermost/bin/mattermost``) to change the ownership of the binary file to *mattermost* user and grant write access:
```
Comment (amyblais) [inline source/install/ee-install.rst:97]:
```suggestion
Open the command line tool on the Mattermost server and ``cd`` to the Mattermost installation directory. Run the following commands (replacing ``<PathToBinaryFile>`` with the appropriate path (typically ``/opt/mattermost/bin/mattermost``)) to change the ownership of the binary file to *mattermost* user and grant write access:
```

File: source/install/ee-install.rst (hunk 11 above)
Comment (justinegeffen) [reply source/install/ee-install.rst]: Thanks, I've revised that and it's fixed. :)

Comment (amyblais) [reply source/install/ee-install.rst]: It seems the bullet point is still misaligned here.

**Comment by amyblais** [comment]
I will go ahead and merge this PR since v5.27 is released.

Please review and open a new PR for any needed edits,

**Pull request merged**